api/events/scheduler.js

```diff
@@ -14,26 +15,28 @@ exports.resetScheduledEvents = () => {
   scheduledEvents = {}
 }
 
-exports.start = function() {
-  return Event.find().then(docs => {
+exports.start = wrap(function *() {
+  try {
+    let docs = yield Event.find()
     const skip = docs.length === 0
     if (skip) return
-    return Promise.all(docs.map(this.create))
-  }).
-  catch(err => log.info(`scheduler failed to start ${err}`))
-}
+    yield Promise.all(docs.map(this.create))
+  } catch (err) {
+    log.info(`scheduler failed to start ${err}`)
```

Comment:
instead using an external logger, you can use directly the Hapi logger, just calling: 
```javascript
server.log('info', `scheduler failed to start ${err}`);
```